### ISSUE TITLE
Support no GeoIP data as default

### ISSUE THREAD

**Issue by softsimon** [mempool/mempool#2553]
Backend should seamlessly work with no GEOIP database set. Not throw an error by default due to hard coded paths (remove hard coded default setting?).

`ERR: <lightning> $lookupNodeLocation() error: ENOENT: no such file or directory, open '/usr/local/share/GeoIP/GeoIP2-ISP.mmdb'`

**Comment by wiz** [comment]
No, the GeoIP database is required for lightning. We can legally redistribute it in the Docker builds for this purpose.

**Comment by softsimon** [comment]
Fix:

Support GeoLite, with graceful fallback.
